<?xml version="1.0"?>
<launch>
<!--	<arg name="vesc_config" default="$(find boat_controller)/config/vesc.yaml" />
	<rosparam file="$(arg vesc_config)" command="load" ns="/"/>
-->
	<node pkg="boat_controller" type="pid_controller.py" name="pid_controller" output="screen">
		  <param name="yawKp" value="0.2" type="double"/>
		  <param name="yawKi" value="0.0" type="double"/>
		  <param name="yawKd" value="0.0" type="double"/>

		  <param name="velKp" value="0.7" type="double"/>
		  <param name="velKi" value="0.3" type="double"/>
		  <param name="velKd" value="0.0" type="double"/>

		  <param name="start_engaged" value="true" type="bool"/>
		  <param name="yaw_cntrl" value="true" type="bool"/>
		  <param name="vel_cntrl" value="true" type="bool"/>

	</node> 

<!--	<node pkg="boat_controller" type="throttle_interpolator.py" name="throttle_interpolator">
		
	</node>
-->
</launch>
